standalone-metastore/metastore-server/pom.xml

```diff
@@ -737,8 +737,8 @@
         <configuration>
           <api>JDO</api>
           <verbose>false</verbose>
-          <log4jConfiguration>${basedir}/src/main/resources/datanucleus-log4j.properties
-          </log4jConfiguration>
+          <log4j2Configuration>${basedir}/src/main/resources/datanucleus-log4j2.xml
```

Comment:
Made this change based on https://www.datanucleus.org/products/accessplatform_6_0/jdo/enhancer.html